code/game/objects/items/devices/scanners.dm

```diff
@@ -264,6 +264,9 @@ REAGENT SCANNER
 	else
 		to_chat(user, "<span class='notice'>Subject's genes are stable.</span>")
 
+	if(HAS_TRAIT(H, TRAIT_HUSK))
+		to_chat(user, "<span class='danger'>Subject is husked. Application of synthflesh is recommended. </span>")
```

Comment:
I'd move this message up a little bit, so that it's around the same place as the broken bone and internal bleeding messages rather than right at the bottom.
![image](https://user-images.githubusercontent.com/57483089/125522654-5bc7752a-fae5-4232-b45b-265ead3ad66d.png)

Reply:
I felt like it was fitting to place it along side where the gene stability stuff is. If a maint/head wants it moved I will do so.